tests/system/test_transfer_manager.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+
+from google.cloud.storage import transfer_manager
+
+from google.api_core import exceptions
+
+
+def test_upload_many(shared_bucket, file_data, blobs_to_delete):
```

Comment:
In a subsequent PR please add a file object of 16mib range so it tests resumable upload code path